### PR TITLE
Blog post - Alicja Suska - 'What happens when a successful product gets its first design team...'

### PR DESCRIPTION
**Pull request by AlicjaSuska**
This version doesn't contain the pictures and the video. They will be added later.

### CODE REVIEW

File: blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+authorUrl: https://alicjasuska.com
+publishDate: 2020-09-11T18:00-07:00
+tags: [blog]
+slug: what-happens-when-a-successful-product-gets-its-first-design-team
+published: true
+---
+
+We are happy to announce that the Sourcegraph design team has grown from 0 to 3! 🎉
+
+The early stages of a new team are full of changes and unexpected challenges. I would like to share with you how our first months of working together look like, what we have accomplished, and what are our plans for the future.
```
Comment (kghopson) [inline blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md:13]:
```suggestion
The early stages of a new team are full of changes and unexpected challenges. I would like to show you what our first months of working together looked like, what we have accomplished, and our plans for the future.
```

File: blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## First 90 days
+
+From the very beginning, we knew that to set our team up for success, we need to carefully plan the next 3 months. There was a lot to focus on: processes, design debt, new initiatives, etc. We had to choose which areas to take care of first and where we can have the biggest impact.
```
Comment (kghopson) [inline blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md:17]:
```suggestion
From the very beginning, we knew that we needed to carefully plan the next 3 months to set our team up for success. There was a lot to focus on: processes, design debt, new initiatives, etc. We had to choose which areas to take care of first and where we could have the biggest impact.
```

**Review by kghopson — APPROVED** [review]
Minor changes in tense and structure. This is a really informative post!

**Comment by AlicjaSuska** [comment]
Thank you @kghopson !

**Review by sqs — APPROVED** [review]

File: blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+---
+title: "What happens when a successful product gets its first design team..."
+author: Alicja Suska
+authorUrl: https://alicjasuska.com
+publishDate: 2020-09-11T18:00-07:00
```
Comment (sqs) [inline blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md:5]: change publishDate to today's date

Comment (AlicjaSuska) [reply blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md]: Changed

**Pull request merged**